### PR TITLE
TopBar implementation

### PR DESCRIPTION
**Pull request by dzucconi**
Will hand over to @csherron for Q/A inside of storybooks before putting this in anywhere. Is fine to merge for now/not being used yet.

### CODE REVIEW

File: react/components/TopBar/components/PrimarySearch/components/PrimarySearchResults/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            );
+          }
+
+          const { results } = searches;
+          const selected = cursor && mod(cursor, results.length + 1);
+
+          this.selectResult(results[selected]);
+
+          return (
+            <React.Fragment>
```
Comment (broskoski) [inline react/components/TopBar/components/PrimarySearch/components/PrimarySearchResults/index.js]: Interestingggg

Comment (dzucconi) [reply react/components/TopBar/components/PrimarySearch/components/PrimarySearchResults/index.js]: what's interesting?

File: react/components/UI/SearchInput/index.js [hunk 2]
```diff
@@ -128,6 +142,10 @@ class SearchInput extends Component {
           onBlur={onBlur}
           onChange={this.handleChange}
           defaultValue={query}
+          autoCorrect="off"
```
Comment (broskoski) [inline react/components/UI/SearchInput/index.js]: Thank you

File: react/components/TopBar/components/PrimarySearch/components/PrimarySearchResults/index.js (hunk 1 above)
Comment (broskoski) [reply react/components/TopBar/components/PrimarySearch/components/PrimarySearchResults/index.js]: Have not seen this before

Comment (dzucconi) [reply react/components/TopBar/components/PrimarySearch/components/PrimarySearchResults/index.js]: Ah, it's just so you can return a single child without an extra DOM element, like wrapping something in a <div>

**Pull request closed without merging**